pixelish/usr/share/asteroid-launcher/watchfaces/pixelish.qml

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright (C) 2021 - Timo Könnecke <github.com/eLtMosen>
+ * Copyright (C) 2022 - Commenter25 <github.com/Commenter25>
+ *
+ * All rights reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 2.1 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+ 
+/* based off digital-shifted */
+
+import QtQuick 2.9
+import QtGraphicalEffects 1.15
+
+Item {
+	id: root
+	
+	property real arcEnd: wallClock.time.getSeconds() * 6
+	onArcEndChanged: canvas.requestPaint()
+
+	/* Uncomment this block for smooth seconds 
```

Comment:
Commented code blogs are not perfectly clean and for stock watchfaces, i'd request to change that. But for communiyt watchfaces its fine imo. Maybe you or someone else can improve this feature later on.

Reply:
The only reason I really did that is because, as far as I'm aware, you can't expose options for a watchface in the GUI. Figure if anyone wants it, they can just uncomment that block. I don't mind removing it, I doubt many would find it. I personally think it looks better ticking anyway, though subjectivity is subjective.